packages/find-root/src/index.ts

```diff
@@ -115,7 +115,7 @@ export async function findRoot(
   let rootDir = await findUp(
     async (directory) => {
       try {
-        await fs.access(path.join(directory, "package.json"));
+        await fs.promises.access(path.join(directory, "package.json"));
```

Comment:
let's prefer using `fs/promises` throughout this PR

Reply:
done